plugins/woocommerce/tests/php/src/Blocks/StoreApi/Routes/Products.php

```diff
@@ -66,12 +67,31 @@ public function test_get_item() {
 		$this->assertEquals( $this->products[0]->add_to_cart_text(), $data['add_to_cart']->text );
 		$this->assertEquals( $this->products[0]->add_to_cart_description(), $data['add_to_cart']->description );
 		$this->assertEquals( $this->products[0]->is_on_sale(), $data['on_sale'] );
+		$this->assertCount( 0, $data['grouped_products'] );
 
 		$this->assertCount( 1, $data['images'] );
 		$this->assertIsObject( $data['images'][0] );
 		$this->assertEquals( $this->products[0]->get_image_id(), $data['images'][0]->id );
 	}
 
+	/**
+	 * Test get grouped product.
+	 */
+	public function test_grouped_product() {
+		$response = rest_get_server()->dispatch( new \WP_REST_Request( 'GET', '/wc/store/v1/products/' . $this->products[2]->get_id() ) );
+		$data     = $response->get_data();
+
+		$grouped_product_ids = $this->products[2]->get_visible_child_ids();
```

Comment:
This test will need to be updated now that `get_visible_child_ids()` has been removed.